src/mozanalysis/experiment.py

```diff
@@ -313,29 +341,35 @@ def build_query(
         )
 
         return """
-    WITH analysis_windows AS (
-        {analysis_windows_query}
-    ),
-    raw_enrollments AS ({enrollments_query}),
-    segmented_enrollments AS ({segments_query}),
-    enrollments AS (
+    CREATE TEMPORARY TABLE enrollments AS (
+        WITH analysis_windows AS (
+            {analysis_windows_query}
+        ),
+        raw_enrollments AS ({enrollments_query}),
+        segmented_enrollments AS ({segments_query}),
+        enrollments AS (
+            SELECT
+                e.*,
+                aw.*
+            FROM segmented_enrollments e
+            CROSS JOIN analysis_windows aw
+        )
+    );
+
+    CREATE OR REPLACE TABLE {destination_table} AS (
```

Comment:
Is `CREATE OR REPLACE` a good idea? If this table is being replaced rather than created, is that a sign that something is going wrong (i.e. we're not caching properly)?

Or would that be "too clever" and bite us in the bum when we're trying to re-run mozanalysis after a failed ETL run?

Reply:
This is useful for Jetstream because we re-use names and don't want the caching behavior.